axelor-account/src/main/java/com/axelor/apps/account/service/debtrecovery/DebtRecoveryService.java

```diff
@@ -279,9 +287,15 @@ public LocalDate getReferenceDate(DebtRecovery debtRecovery) {
             }
           }
         }
-        // échéances rejetées qui ne sont pas bloqués
-        else if (move.getInvoice() == null) {
-          if (moveLine.getPaymentScheduleLine() != null
+        // Invoice term ou échéances rejetées qui ne sont pas bloqués
```

Comment:
Please write this comment in english (or maybe simply remove it if it does not help to understand the method)